<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmIpcEditDialog</class>
 <widget class="QDialog" name="frmIpcEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>操作员登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetTitle" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labIco">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labTitle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>编辑摄像机信息</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMenu_Close">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetMain" native="true">
     <property name="styleSheet">
      <string notr="true">#gboxSoftKey .QPushButton{
border-width:0px;
border-radius:0px;
padding:2px;
}

#gboxSoftKey .QPushButton:hover{
border:1px solid #FF0000;
}

#gboxSoftKey{
border-width:0px;
border-radius:0px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="4">
          <widget class="QWidget" name="widget_2" native="true">
           <widget class="QLabel" name="labelID">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>24</y>
              <width>18</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>ID:</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEditID">
            <property name="geometry">
             <rect>
              <x>69</x>
              <y>24</y>
              <width>61</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEditName">
            <property name="geometry">
             <rect>
              <x>69</x>
              <y>65</y>
              <width>133</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="labelName">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>65</y>
              <width>54</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>叠加名称:</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEditIP">
            <property name="geometry">
             <rect>
              <x>69</x>
              <y>106</y>
              <width>133</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="labelIP">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>106</y>
              <width>42</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>IP地址:</string>
            </property>
           </widget>
           <widget class="QLabel" name="labelUsrName">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>147</y>
              <width>42</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>用户名:</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEditUsrName">
            <property name="geometry">
             <rect>
              <x>69</x>
              <y>147</y>
              <width>133</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="labelType">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>150</y>
              <width>30</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>类型:</string>
            </property>
           </widget>
           <widget class="QComboBox" name="cbBoxType">
            <property name="geometry">
             <rect>
              <x>330</x>
              <y>150</y>
              <width>69</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEditPwd">
            <property name="geometry">
             <rect>
              <x>69</x>
              <y>188</y>
              <width>133</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="labelPwd">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>188</y>
              <width>30</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>密码:</string>
            </property>
           </widget>
           <widget class="QLabel" name="labelDetect">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>190</y>
              <width>54</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>移动侦测:</string>
            </property>
           </widget>
           <widget class="QComboBox" name="cbBoxDetect">
            <property name="geometry">
             <rect>
              <x>330</x>
              <y>190</y>
              <width>69</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="labelMainStream">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>229</y>
              <width>42</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>主码流:</string>
            </property>
           </widget>
           <widget class="QComboBox" name="cbBoxMainStream">
            <property name="geometry">
             <rect>
              <x>69</x>
              <y>229</y>
              <width>69</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="labelSubStream">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>230</y>
              <width>42</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>子码流:</string>
            </property>
           </widget>
           <widget class="QComboBox" name="cbBoxSubStream">
            <property name="geometry">
             <rect>
              <x>330</x>
              <y>230</y>
              <width>69</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="labelPTZ">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>270</y>
              <width>54</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>云台地址:</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEditPTZ">
            <property name="geometry">
             <rect>
              <x>69</x>
              <y>270</y>
              <width>251</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="btnCancle">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="btnConfirm">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
